docs/configuration/selecting-excluding.rst

```diff
@@ -0,0 +1,43 @@
+.. _selecting-excluding:
+
+Selecting & Excluding
+=======================
+
+Cosmos allows you to filter by configs (e.g. ``materialized``, ``tags``) using the ``select`` and ``exclude`` parameters. If a model contains any of the configs in the ``select``, it gets included as part of the DAG/Task Group. Similarly, if a model contains any of the configs in the ``exclude``, it gets excluded from the DAG/Task Group.
+
+The ``select`` and ``exclude`` parameters are dictionaries with the following keys:
```

Comment:
Since #360, they are no longer dictionaries - they are lists.
An example of the syntax:
```
select=["config.materialized:table"]
```

More examples can be found at:
https://github.com/astronomer/astronomer-cosmos/blob/main/tests/dbt/test_selector.py